software/control/gui_hcs.py

```diff
@@ -301,9 +300,8 @@ def loadObjects(self, is_simulation):
                 self.camera_focus,
                 self.liveController_focus_camera,
                 self.stage,
-                has_two_interfaces=HAS_TWO_INTERFACES,
-                use_glass_top=USE_GLASS_TOP,
-                look_for_cache=False,
+                self.objectiveStore,
+                self.acquisitionConfigurationManager.get_autofocus_configurations()
```

Comment:
(mark this to look at - laserAutofocusController probably should not access self.acquisitionConfigurationManager)